x-pack/plugin/core/src/main/resources/fleet-filedelivery-data.json

```diff
@@ -0,0 +1,42 @@
+{
+  "index_patterns": [
+    ".fleet-filedelivery-data-*-*"
```

Comment:
If we decide to move to data streams, these indices should be converted as well.